_drafts/STMViewer.md

```diff
@@ -0,0 +1,161 @@
+---
+# This is a template for Interrupt posts. Use previous, more recent posts from the _posts/
+# directory for more inspiration and patterns.
+#
+# When submitting a post for publishing, submit a PR with the post in the _drafts/ directory
+# and it will be moved to _posts/ on the date of publish.
+#
+# e.g.
+# $ cp _drafts/_template.md _drafts/my_post.md
+#
+# It will now show up in the front page when running Jekyll locally.
+
+title: Visualizing embedded data made easy
+description: 
+    "A brief introduction to STMViewer software use cases in debugging embedded targets"
+author: piotrw
+---
+
+<!-- excerpt start -->
+
+If you've ever wanted to plot data acquired on your embedded target, this article is for you. It explores common use cases for real-time data visualization using STMViewer. Say goodbye to manual, time-consuming, and error-prone data collection and display methods to speed up your debugging process.
+
+<!-- excerpt end -->
+
+{% include newsletter.html %}
+
+{% include toc.html %}
+
+## What exactly can it visualize?
```

Comment:
I think it might be helpful to walk through why someone would want to do this. What kind of problem would it help you solve? Right now we have a good look at some tools, but the motivation is unclear.